src/Robo/Plugin/Commands/ThemeCommands.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace ChqRobo\Robo\Plugin\Commands;
+
+use Robo\Exception\TaskException;
+use Robo\Result;
+use Robo\Robo;
+use Robo\Tasks;
+
+/**
+ * Robo commands related to theme operations.
+ */
+class ThemeCommands extends Tasks
```

Comment:
Are we prototyping these for CHQ, then planning to move them to our shared repo? I haven't been following this very closely.

Reply:
@adamzimmermann This is the public repo.

Reply:
🤦🏼 I saw `chq` in the repo name and clearly quit reading after that.